src/Access/AccessControlManager.cpp

```diff
@@ -32,6 +41,110 @@ namespace
 
     constexpr size_t DISK_ACCESS_STORAGE_INDEX = 0;
     constexpr size_t USERS_CONFIG_ACCESS_STORAGE_INDEX = 1;
+
+    auto parseLDAPServer(const Poco::Util::AbstractConfiguration & config, const String & ldap_server_name)
```

Comment:
Let's move these parsing details into class `ExternalAuthenticators`, it doesn't look very good here.

Reply:
Ok, will do.

Reply:
Done.